sites/svelte-5-preview/src/routes/docs/content/01-api/05-imports.md

```diff
@@ -115,6 +115,39 @@ Svelte provides reactive `Map`, `Set`, `Date` and `URL` classes. These can be im
 <input bind:value={url.href} />
 ```
 
+## `svelte/events`
+
+Svelte provides a way of imperatively attaching DOM event listeners to elements using the `attach` export from `svelte/events`. This can be used in place of
```

Comment:
Thanks for this everyone. Pardon my ignorance, and I am sure there must be a reason that `attach` is introduced, but i think more explanation is needed here, for why we need an additional abstraction layer instead of the primitive `addEventListener`, especially when this wasn't a problem in Svelte 4 and will probably catch people off guard.

I've always advocated that Svelte tries to stay very closely to the core web APIs, and I can already imagine people saying "is there really no way to make `addEventListener` work?".

Reply:
Svelte 5 has it's delegated event system to improve performance, just like how other frameworks do the same. This means that things like `addEventListener` can have issues when co-ordinating with Svelte's event delegation system. Svelte 4 didn't have delegated events, but evidence has been shown that delegating events can have a significant impact in large applications and also on startup. I can tune the messaging above though.

Reply:
Thanks Dominic, much appreciated.

Reply:
I updated the docs to include a bit about it.